### ISSUE TITLE
Partially-broken PornHub support

### ISSUE THREAD

**Issue by Knocks83** [ytdl-org/youtube-dl#28393]
## Checklist

- [X] I'm reporting a broken site support
- [X] I've verified that I'm running youtube-dl version **2021.03.03**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

## Description

Basically PornHub has removed their included download feature as per their ["Commitment to Trust and Safety"](https://help.pornhub.com/hc/en-us/categories/360002934613) part 2.
The only links affected are the ones using this format: `https://www.pornhub.com/video/get_media?s=...`. If you download using the standard settings it'll download just fine.

**Comment by Rogue-Git-Dev** [comment]
Today I noticed almost no downloads work. YouTube-DL saved a M3U8 playlist that indexes segments of TS streams. I know they are TS streams because another programme I tried loaded the TS segments instead of the actual MP4 file. They definitely changed something globally not for the better. YouTube-DL even told me randomly after trying for a while to instal PhantomJS which is not supported upstream. This was very peculiar.
